infra/terraform/modules/data_upload_user/outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "system_user_access_key_id" {
```

Comment:
Minor point here. I'd keep these output names shorter and just rename them:
- `system_user_access_key_id` => `access_key_id`
- `system_user_access_key_secret` => `access_key_secret`

The fact these are for a user is kind of implied by the fact this is in the 'data_upload_**user**' module.

Reply:
good point - changed